README.md

```diff
@@ -45,3 +48,23 @@ trigger the [scope
 inspector](https://code.visualstudio.com/api/language-extensions/syntax-highlight-guide#scope-inspector)
 from the Command Palette (`Ctrl+Shift+P`) with the `Developer: Inspect Editor
 Tokens and Scopes` command.
+
+### User Guide 
```

Comment:
```suggestion
## User Guide 
```
This would put the user guide at the same level as "Development workflow", rather than a subsection of it.